getting-started/ts-example/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "plugins": ["@typescript-eslint", "node"],
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  "parser": "@typescript-eslint/parser",
+  "rules": {
+    "@typescript-eslint/no-var-requires": 0,
+    "import/prefer-default-export": "off",
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
```

Comment:
what does this do ?

Reply:
It causes files with extension `""` (no extension) to throw lint errors. For example importing `./foo.js` is ok but `./foo` is not. `ignorePackages` prevents the error from being thrown when importing packages (`./foo` is allowed if importing `./foo/index.ts`).